README.md

```diff
@@ -63,7 +63,7 @@ dotnet build
 cd tools
 
 # Instrument assemblies inside 'test' folder to detect hits for source files inside 'src' folder
-dotnet minicover instrument --workdir ../ --assemblies test/**/bin/**/*.dll --sources src/**/*.cs 
+dotnet minicover instrument --configdir ~/out --workdir ../ --assemblies test/**/bin/**/*.dll --sources src/**/*.cs 
```

Comment:
it looks like breaking changes, not like another option to be invoked.

Reply:
I'm not sure I follow, can you please explain? 
Thanks!

Reply:
It is the new parameter which is necessary instead optional. It is what provides breaking changes. Other developers which are using MiniCover should changes initialization in every project.

Reply:
I see, I can try to change it to an optional param and have the value fallback to workdir when it's not used.
Does that sound good to you?

Reply:
yes